netbout-client/pom.xml

```diff
@@ -189,6 +189,14 @@
                                  Findbugs filters in current version of Qulice
                                 -->
                                 <exclude>findbugs:~com.netbout.client.*</exclude>
+                                <!--
+                                @todo #1013:30min Fix findbugs errors in mock
```

Comment:
@kujtimiihoxha please make it proper text, with periods at the end of sentences and spaces after periods :)